packages/accounts/src/models/ilpAccount.ts

```diff
@@ -3,12 +3,32 @@ import { IlpHttpToken } from './ilpHttpToken'
 import { uuidToBigInt } from '../utils'
 import { Model, Pojo, raw } from 'objection'
 
+function bigIntToUuid(id: bigint): Pojo {
```

Comment:
Not directly related to this PR, but why ever represent UUIDs as bigints (rather than strings) at all?

Reply:
TigerBeetle account ids are 128-bit.
[tigerbeetle-node](https://github.com/coilhq/tigerbeetle-node/blob/main/src/index.ts#L23) deals with these as `bigint`s.
UUIDs seems to be the best way to store 128-bit values in postgres (https://github.com/coilhq/rafiki/pull/18#discussion_r649345413)

Reply:
Ok. I wasn't referring to how the are stored in postgres -- I was talking about how they are represented on the model.

> tigerbeetle-node deals with these as bigints.

That answers my question.

Reply:
Great question @sentientwaffle.

Just to add some background explanation on why `tigerbeetle-node` uses a JS BigInt rather than a JS String type:

I believe a String in V8 requires at least 24 bytes to store regardless of how small the string is, so there is more "boxing" overhead for String metadata (at least this was the case when last I checked a few years ago). It's also stored as a form of UTF-8 I think so there may be decoding overhead to get at the raw bytes... I'm not sure.

A BigInt might have a little bit of boxing overhead to store the word count but should be more efficient. However, we haven't dived into the V8 source to double-check this.

Apart from these smaller details, the main reason is to keep the fields all as first-class numbers, which is how they are represented in the Zig client (`u128`s).